libs/dal/src/repositories/organization/organization.repository.ts

```diff
@@ -12,25 +12,24 @@ export class OrganizationRepository extends BaseRepository<OrganizationDBModel,
   }
 
   async findById(id: string, select?: string): Promise<OrganizationEntity | null> {
-    const data = await this.MongooseModel.findById(id, select);
+    const data = await this.MongooseModel.findById(id, select).read('secondaryPreferred');
     if (!data) return null;
 
     return this.mapEntity(data.toObject());
   }
 
-  async findOrganizationById(organizationId: string): Promise<OrganizationEntity | null> {
-    const data = await this.MongooseModel.findById(organizationId).read('secondaryPreferred');
-    if (!data) return null;
+  async findUserActiveOrganizations(userId: string): Promise<OrganizationEntity[]> {
+    const organizationIds = await this.getUsersMembersOrganizationIds(userId);
 
-    return this.mapEntity(data.toObject());
```

Comment:
@scopsy do you remember why we created new `findOrganizationById` that reads from secondaryPreferred for the widget API route? instead of reusing the existing `findById` and updating its query to use secondaryPreferred?

At the moment I do not see reasons for duplication, and all `findById` can use secondaryPreferred.

Reply:
Sounds good